src/site/markdown/server/manage-webadmin.md

```diff
@@ -2816,6 +2817,29 @@ while:
  - userExportFrom: export deleted messages from this user
  - exportTo: content of deleted messages have been shared to this mail address
  - totalExportedMessages: number of deleted messages match with json query, then being shared to sharee
+ 
+### Purge Deleted Messages
+ 
+First, you need to configure 'retentionPeriod' in deletedMessageVault.
```

Comment:
what if it's not the case? I don't see any error specific to that? is it HTTP 500?

Reply:
It will use default value (1 year)